app/scripts/browser.d.ts

```diff
@@ -0,0 +1 @@
+declare const browser: typeof chrome
```

Comment:
Oh interesting - does this prevent TS warnings?

Reply:
It can help my vscode to understand the shape of `browser` and make some suggestions.
![image](https://user-images.githubusercontent.com/14259049/110057670-f7756180-7d9b-11eb-8273-fb511fb6cdab.png)

And thanks to [webextension-polyfill](https://www.npmjs.com/package/webextension-polyfill#usage-with-typescript), some api's shape has differences.

```js
// native chrome api will call a callback when async query done
broswer.tabs.executeScript(tabId: number, details: InjectDetails, callback?: (result: any[]) => void): void;

// webextension polyfilled api returns a promise
broswer.tabs.executeScript(tabId: number, details: InjectDetails): Promise<any[]>;
```